### PR TITLE
pull scrape interval from prometheus

### PR DESCRIPTION
**Pull request by AjayTripathy**

### CODE REVIEW

File: pkg/costmodel/promparsers.go [hunk 1]
```diff
@@ -8,8 +8,30 @@ import (
...
+}
+
+func GetPrometheusConfig(pcfg string) (PromCfg, error) {
+	var promCfg PromCfg
+	err := yaml.Unmarshal([]byte(pcfg), &promCfg)
+	return promCfg, err
+}
+
+func GetKubecostJobName() string {
+	return DEFAULT_KUBECOST_JOB_NAME // TODO: look this up from a prometheus variable?
```
Comment (nikovacevic) [inline pkg/costmodel/promparsers.go:32]: So, until we do this, someone with a different job name would get the default of "1m", right?

**Review by nikovacevic — APPROVED** [review]
Makes sense to me. Just that one quick question about behavior when the job name isn't the default job name.

**Comment by nikovacevic** [comment]
Also, do we have an env in which to test weird scrape intervals?

File: pkg/costmodel/promparsers.go (hunk 1 above)
Comment (AjayTripathy) [reply pkg/costmodel/promparsers.go:32]: Yeah. That seem right to you?

**Pull request merged**